ec2/sc-ec2-linux-jumpcloud-workflows-v1.0.0.yaml

```diff
@@ -448,3 +452,5 @@ Outputs:
     Value: !Ref 'InstanceProfile'
   ConnectionInstructions:
     Value: "https://sagebionetworks.jira.com/wiki/spaces/IT/pages/996376579/Connect+to+Provisioned+Instances"
+  ConnectionURI:
```

Comment:
i believe this is already done in PR https://github.com/Sage-Bionetworks/service-catalog-library/pull/156

Reply:
agreed- let's leave those changes to PR #156